frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContext.ts

```diff
@@ -3,6 +3,7 @@ import { UseDisclosureReturn } from '@chakra-ui/react'
 
 type BuilderAndDesignContextProps = {
   mobileCreateEditModal: UseDisclosureReturn
+  deleteFieldModal: UseDisclosureReturn
```

Comment:
nit: rename to a clearer variable such as `xxxModalDisclosure`?

Reply:
[b26f1a0](https://github.com/opengovsg/FormSG/pull/3694/commits/b26f1a0aac9bf5bafec7f27c64eb20334f0cbdfe)